src/feature/stats/rephist.c

```diff
@@ -2455,6 +2455,20 @@ rep_hist_note_circuit_handshake_assigned(uint16_t type)
     onion_handshakes_assigned[type]++;
 }
 
+/** Get the circuit handshake value that is requested. */
+MOCK_IMPL(int,
+rep_hist_get_circuit_handshake_requested, (uint16_t type))
+{
```

Comment:
We should be extra extra careful here with `type` so I would recommend you `BUG()` on `type > MAX_ONION_HANDSHAKE_TYPE` and return 0 as in "no stats".